src/solvers/dg_p4est/containers.jl

```diff
@@ -1,5 +1,5 @@
 # Note: This is an experimental feature and may be changed in future releases without notice.
```

Comment:
I didn't check this file's changes in detail... I assume that all changes in here are related to the fact that we now use resizable `Vector` for the underlying storage, allow resizing of the data raw structures, and reinitialization of the containers.

If there were other changes or if I should do a more in-depth review, please let me know.

Reply:
Yes, the only changes here are
- changing all data structures to use `unsafe_wrap`ped `Vector`s internally,
- adding `reinitialize_containers!`,
- replacing some `convert_sc_array` with `Tuple`s,
- adding `load_sc_array`.

Reply:
Maybe add some more documentation to `load_sc_array` (see also my other comment before). Also, I am not sure if I like the `i=1` or whether it would be better to always explicitly pass `1` to indicate that one is accessing the first element of an array-like container.

Reply:
The `i=1` is similar to how `unsafe_load` works.

Reply:
Ah, now I get the naming conventions! Maybe then call it `unsafe_load_sc`? Argh, so many names, I dunno... I just would like to drop the `array` from the name part to avoid having people assume it's loading an array (instead of a value).